ddtrace/contrib/celery/task.py

```diff
@@ -48,6 +55,13 @@ def patch_task(task, pin=None):
     pin.onto(task)
     return task
 
+def patch_task_with_pin(pin=None):
```

Comment:
If it's used only in tests, we should move it in our test utils (inside `tests/contrib/celery`) otherwise developers may use it. Here we should keep only the public API.